lib/helpers/ssh_client.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+require_relative '../management'
+
+# Class for interaction with net/ssh
+# https://www.rubydoc.info/github/net-ssh/net-ssh/Net/SSH
+class SshClient
+  # Initializing ssh connection method
+  # @param [String] host remote server host
+  # @param [String] user user on whose behalf the access will be granted
+  # @param [Object] options The options are described in the module documentation
+  # @param [Proc] block Instructions for interacting with an ssh connection.
+  #                     The connection is closed when the block is executed.
+  # @return [Array, Net::SSH::Authentication] https://www.rubydoc.info/github/net-ssh/net-ssh/Net/SSH/Authentication
+  def connect(host, user, options, &block)
+    Net::SSH.start(host, user, options, &block)
+  end
+
+  class << self
```

Comment:
I think all those methods can be instance method, not class method, since you still send instance via `session` param in both class methods